README.md

```diff
@@ -65,17 +65,21 @@ class { 'self_service':
-| S0005        | Determines if CA expires within 90 days                     | Install the following module https://forge.puppet.com/modules/puppetlabs/ca_extend and follow the documentation to extend the CA               |                                                                                                                                                                                                                                        |
-| S0006        | Determines if 15m Load Average is greater than available cores |                                                                                                                                                 |                                                                                                                                                                                                                                        |
-| S0007        | Determines if there is at least 20% disk free on postgres Data partition|                                                                                                                                                |                                                                                                                                                                                                                                        |
-| S0008        |                                                             |                                                                                                                                                |                                                                                                                                                                                                                                        |
-
+| Indicator ID | Self Service Steps                                                                 | What to Include in a Support Ticket                                                                                                                                                                                                                                                                                                                                                                                                                                         |                                                                                                                                                                                                                                       |
```

Comment:
The second column is missing a title, resulting in the column titles being off and the last one missing a title

Reply:
DOH!